src/modelbench/static_site_generator.py

```diff
@@ -136,6 +136,7 @@ def content_sut(self, item: SutDescription, key: str):
         if item.key in self._content:
             return self._content[item.key][key]
         else:
+            return f"{key} ({item})"
```

Comment:
The `raise` below is unreachable now. Should we get rid of it?

Reply:
Ah, right. I specifically wanted to ask @dhosterman about this. For SUTs that we control, we'll put in friendly names. For others, I'm not sure how to handle the in-report name. This change keeps things from catching on fire, but it's not great otherwise. Any ideas?

Reply:
Since Daniel's out, I've change this to warn about the issue, as well as to do something reasonable when it's a name being looked for.